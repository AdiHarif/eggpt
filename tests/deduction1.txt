Wolves are afraid of mice.
Sheep are afraid of mice.
Winona is a sheep.
Mice are afraid of cats.
Cats are afraid of wolves.
Jessica is a mouse.
Emily is a cat.
Gertrude is a wolf.
What is emily afraid of?
What is winona afraid of?
What is gertrude afraid of?	
What is jessica afraid of?
