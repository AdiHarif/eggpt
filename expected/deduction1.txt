What is emily afraid of?	wolf	7 5
What is winona afraid of?	mouse	3 2
What is gertrude afraid of?	mouse	8 1
What is jessica afraid of?	cat	6 4
